src/notation/inotation.h

```diff
@@ -72,6 +72,9 @@ class INotation
     // styles
     virtual INotationStylePtr style() const = 0;
 
+    // score
+    virtual Ms::Score* score() const = 0;
+
```

Comment:
This change seems unnecessary

Reply:
Removing this code causes my build to crash. Is it the same for you too?

Reply:
No, that sounds strange... I'll investigate tomorrow.